eq-author/src/components/CollapsibleNavItem/index.js

```diff
@@ -37,9 +37,13 @@ const Button = styled.button`
   overflow: hidden;
   width: 100%;
 
-  border-top: ${({ bordered }) => (bordered ? `1px solid ${colors.grey}` : "")};
-  border-bottom: ${({ bordered }) =>
-    bordered ? `1px solid ${colors.grey}` : ""};
+  ${({ bordered, borderColor }) =>
+    bordered
+      ? `
+  border-bottom: 1px solid ${borderColor};
+  border-top: 1px solid ${borderColor};
+`
+      : ""}
```

Comment:
Would this work? 

```suggestion
  ${({ bordered, borderColor }) =>
    bordered ?? `
        border-bottom: 1px solid ${borderColor};
        border-top: 1px solid ${borderColor};
    `
}
```

Reply:
That would just insert whatever was in `bordered` (if any value other than `undefined` or `null`) into the CSS